lib/improver/tests/nbhood/test_squareneighbourhood.py

```diff
@@ -198,12 +198,24 @@ def test_multiple_times(self):
             cube, self.width, self.width)
         self.assertArrayAlmostEqual(result.data, expected_result)
 
+    def test_neighbourhood_larger_than_half_domain(self):
+        """Test the calculation of edge columns and rows is ok when the
+           neighbourhood size is larger than half the domain."""
+        expected_result = np.array([[0.9375, 0.95, 0.95, 0.95, 0.9375],
+                                    [0.95, 0.96, 0.96, 0.96, 0.95],
+                                    [0.95, 0.96, 0.96, 0.96, 0.95],
+                                    [0.95, 0.96, 0.96, 0.96, 0.95],
+                                    [0.9375, 0.95, 0.95, 0.95, 0.9375]])
+        result = SquareNeighbourhood().mean_over_neighbourhood(
+            self.cube, 3, 3)
```

Comment:
Can you define a new value called `width = 3` and pass this in instead. I think this will make this test a little clearer.